spec/support/spec/fiberspec.rb

```diff
@@ -2,6 +2,7 @@
 
 shared_examples 'fiberspec' do
   it do
+    is_expected.to contain_class('profile::ts::dco')
```

Comment:
The `support/spec/*` shared examples have be used to test the functionality essentially within a single profile.  They certainly could be used for a role but we haven't been doing that as I think it wouldn't compose very well.  I don't think this shared example should test for "dco" resources in the catalog.